pkg/foxtrot/db_test.go

```diff
@@ -0,0 +1,74 @@
+package foxtrot
+
+import (
+	"context"
+	"errors"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreateGetUser(t *testing.T) {
+	db, err := newDB("")
+	require.NoError(t, err)
+	defer db.close()
+
+	u := &User{Name: "alice", passwordHash: "###"}
+	err = db.createUser(context.Background(), u)
+	require.NoError(t, err)
+
+	u2, err := db.getUser(context.Background(), "alice")
+	require.NoError(t, err)
+	require.Equal(t, u, u2)
+}
+
+func TestGetUserErr(t *testing.T) {
+	db, err := newDB("")
+	require.NoError(t, err)
+	defer db.close()
+
+	_, err = db.getUser(context.Background(), "MISSING")
```

Comment:
Miss Ing will be disappointed to know she is an error.

Reply:
I'm Miss Ing. Dipl-Ing is my legal title for having finished my CS degree (Diplomingeneur).